api/envoy/extensions/filters/http/custom_response/v3/policies.proto

```diff
@@ -0,0 +1,90 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.custom_response.v3;
+
+import "envoy/config/core/v3/base.proto";
+import "envoy/config/core/v3/extension.proto";
+import "envoy/config/core/v3/substitution_format_string.proto";
+
+import "google/protobuf/wrappers.proto";
+
+import "xds/annotations/v3/status.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.custom_response.v3";
+option java_outer_classname = "PoliciesProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/http/custom_response/v3;custom_responsev3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+option (xds.annotations.v3.file_status).work_in_progress = true;
+
+// [#protodoc-title: Custom Response Policies]
+// [#extension: envoy.filters.http.custom_response]
```

Comment:
This seems to be the same name as the CER filter. Probably should be different.

Reply:
The CER filter has the name `Custom Response Filter`

Reply:
Do you think the source and apis for policies should live under `<source/api>/extensions/custom_response/policies` or `<source/api>extensions/http/custom_response/policies`

Reply:
I think your second suggestion (`<source/api>extensions/http/custom_response/policies`) is better, as these policies are only for HTTP responses.

In addition to moving the location, I suggest having a new extension category for the policies (e.g., `envoy.http.custom_response.policies`) and update the CER filter documentation and reference this extension category, so one can know which policies can be used in `on_match` actions.